dali/pipeline/operators/util/optical_flow.cc

```diff
@@ -0,0 +1,89 @@
+// Copyright (c) 2019, NVIDIA CORPORATION. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include <dali/pipeline/data/views.h>
+#include <dali/pipeline/operators/util/optical_flow.h>
+
+namespace dali {
+
+
+DALI_SCHEMA(OpticalFlow)
+                .DocStr(R"code(Calculates the Optical Flow for sequence of images given as a input.
+ Mandatory input for the operator is a sequence of frames.
+ As an optional input, operator accepts external hints for OF calculation.
+The output format of this operator matches the output format of OF driver API.
```

Comment:
Inconstant spaces at the begging of the doc.

Reply:
You could add more detailed reference where to find this Driver API

Reply:
Done